backend/src/main/java/com/bakdata/conquery/util/search/TrieSearch.java

```diff
@@ -119,80 +123,84 @@ private Stream<String> split(String keyword) {
 	}
 
 	/**
-	 * A lower weight implies more relevant words.
+	 * Calculate and update weights for all queried items
 	 */
-	private double weightWord(String keyword, String itemWord) {
+	private void updateWeights(String query, final Map<String, List<T>> items, Object2LongMap<T> itemWeights, boolean original) {
+		if (items == null) {
+			return;
+		}
+
+		for (Map.Entry<String, List<T>> entry : items.entrySet()) {
+			final List<T> hits = entry.getValue();
 
-		// Test if the word is an original word and not a suffix.
-		final boolean isOriginal = isOriginal(itemWord);
+			if (hits.isEmpty()) {
+				continue;
+			}
+
+			long weight = weightWord(query, entry.getKey(), original);
+
+			// We combine hits multiplicative to favor items with multiple hits
+			for (T item : hits) {
+				itemWeights.put(item, itemWeights.getOrDefault(item, 1) + weight);
+			}
```

Comment:
Das sieht mir nach einer Summer anstatt einer Multiplikation aus. Aber das weighting verstehe ich auch nicht so wirklich

Reply:
Wir sollten  auch auf jden Fall multiplizieren, das sieht nach einem typo aus

Reply:
Widersprecht ihr euch gerade?
Fabian -> multiplizieren
Sebastian -> addieren

Reply:
ja, sebastian hatte in der zwischenzeit den code angepasst